ubuntu-personal-store/ubuntu-personal-store.qml

```diff
@@ -65,7 +65,18 @@ ApplicationWindow {
 
     context: WebContext {
         dataPath: main.cookiePath
-        sessionCookieMode: WebContext.SessionCookieModeRestored
+    }
+
+    Component.onCompleted: {
+      var request = new XMLHttpRequest()
+      request.open('GET', context.dataPath + '/token.txt')
+      request.onreadystatechange = function(event) {
+          if (request.readyState == XMLHttpRequest.DONE) {
+              var token = request.responseText
+              context.cookieManager.setCookies(main.url, [{"name": "SM", "value": token}]);
+          }
+      }
+      request.send()
```

Comment:
The request here is asynchronous ... there might be a race w/ the app displaying the content